src/db_backup.js

```diff
@@ -0,0 +1,64 @@
+import {nanoid} from 'nanoid';
+
+const db = [
+	{
+		id: nanoid(),
+		accountName: 'bargeld',
+		bankName: 'Hosentasche',
+		accountValue: 8,
+		entries: [
+			{
+				id: nanoid(),
+				date: '07-06-2022',
+				change: -40.31,
+				note: 'cloth',
+			},
+			{
+				id: nanoid(),
+				date: '08-06-2022',
+				change: 50.0,
+				note: 'visit grandma',
+			},
+		],
+	},
+	{
+		id: nanoid(),
+		accountName: 'Girokonto',
+		bankName: 'coole Bank',
+		accountValue: 700,
+		entries: [
+			{
+				id: nanoid(),
+				date: '23-06-2022',
+				change: -500,
+				note: 'Frau war einkaufen',
+			},
+			{
+				id: nanoid(),
+				date: '09-06-2022',
+				change: 50.0,
+				note: 'visit grandma',
+			},
+		],
+	},
+	{
+		id: nanoid(),
+		accountName: 'Sparbuch',
+		bankName: 'nicht so coole bank',
+		accountValue: 2000,
+		entries: [
+			{
+				id: nanoid(),
+				date: '07-06-2022',
+				change: -300,
+				note: 'new monitor',
+			},
+			{
+				id: nanoid(),
+				date: '08-06-2022',
+				change: 50.0,
+				note: 'visit grandma',
+			},
+		],
+	},
+];
```

Comment:
Safety first, but please delete the backup file.😁

Reply:
The file is deleted :)